FAQ.md

```diff
@@ -84,6 +85,18 @@ RuneLite is updated every couple weeks, usually immediately following the Wednes
 ## Where can I find the logs, screenshots, or configurations?
 To find logs, either open screenshot directory (by right-clicking "Camera" button) and navigate 1 directory up and then open logs folder, or navigate to `%userprofile%\.runelite\logs` on Windows or `$HOME/.runelite/logs` on Linux and macOS.
 
+## How can I change where screenshots get saved?
+It's not possible to directly change where RuneLite saves screenshots. However, you can create a symlink between the screenshots folder and where you actually want them saved.
+
+### On Windows:
+- Move your screenshots folder to where you want to have your screenshots saved, and then delete the `screenshots` folder that's in your `.runelite` folder.
+- Run Command Prompt as administrator and then run:
+
+`mklink /J "%userprofile%\.runelite\screenshots" "path\to\new\folder"`
```

Comment:
If you make this `` ``` `` it'll have a copy and paste option probably. Might also be nice to indent it if it's part of the list, but idk if that's possible with `` ``` ``

Reply:
since youre providing a `mv` command for mac/linux, you should provide the equivalent for windows with `move` here, then the above section isnt really required anymore

Reply:
> since youre providing a `mv` command for mac/linux, you should provide the equivalent for windows with `move` here, then the above section isnt really required anymore

In trying to do so on Windows, I'm actually running into a file permissions error, even with cmd running as administrator. I think it's probably easier for the average end user just to move the files manually rather than having them fiddle with file permissions.

I'm open to still changing this, but I think leaving it as I have it now is probably the most straightforward way of doing it for the typical user.